week-4/D-methods/README.md

```diff
@@ -1,19 +1,69 @@
-### Goals
-- make objects even more useful and more closer to "real-life" by introducing object methods
-- explain how to get access to the current object reference (the `athlete` inside the the `drink` function in the example below)
+### Object methods
 
-The should be able to understand the following:
+Besides having specific properties, objects in the real world can also do things. For example, a plane can fly, a computer can display something on the screen, an athlete can run etc... To translate this into the Javascript world, let's introduce object's methods.
 

```

Comment:
I think it's probably best to avoid examples like "flying", "running" etc, since objects are more anaemic in practice - methods just access and manipulate the data in the object. It'd be rare to implement more complicated functionality which uses the data (e.g displaying it on the screen) in a method on the object itself. Also it might be an idea to link the use of "methods" to the ones we've already seen for arrays.